<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionsViewWidgetClass</class>
 <widget class="QWidget" name="ActionsViewWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">OverViewWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ActionsView" name="treeView">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
    </widget>
   </item>
  </layout>
  <action name="filterTillAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res/res.qrc">
     <normaloff>:/pic/soilfield.png</normaloff>:/pic/soilfield.png</iconset>
   </property>
   <property name="text">
    <string>Tillage</string>
   </property>
   <property name="toolTip">
    <string>Tillage</string>
   </property>
  </action>
  <action name="filterMedwAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res/res.qrc">
     <normaloff>:/pic/meadow.png</normaloff>:/pic/meadow.png</iconset>
   </property>
   <property name="text">
    <string>Pasture</string>
   </property>
   <property name="toolTip">
    <string>Pasture</string>
   </property>
  </action>
  <action name="filterPltrAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res/res.qrc">
     <normaloff>:/pic/pltr32.png</normaloff>:/pic/pltr32.png</iconset>
   </property>
   <property name="text">
    <string>Plant Treatment</string>
   </property>
  </action>
  <action name="filterDrilAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res/res.qrc">
     <normaloff>:/pic/crop32.png</normaloff>:/pic/crop32.png</iconset>
   </property>
   <property name="text">
    <string>Drilling&amp;&amp;Planting</string>
   </property>
   <property name="toolTip">
    <string>Drilling&amp;&amp;Planting</string>
   </property>
  </action>
  <action name="filterHarvAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res/res.qrc">
     <normaloff>:/pic/cut.png</normaloff>:/pic/cut.png</iconset>
   </property>
   <property name="text">
    <string>Harvesting</string>
   </property>
   <property name="toolTip">
    <string>Harvesting</string>
   </property>
  </action>
  <action name="filterFertAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res/res.qrc">
     <normaloff>:/pic/fert.png</normaloff>:/pic/fert.png</iconset>
   </property>
   <property name="text">
    <string>Fertilisation</string>
   </property>
   <property name="toolTip">
    <string>Fertilisation</string>
   </property>
  </action>
  <action name="filterAnimGrpAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res/res.qrc">
     <normaloff>:/pic/cow32.png</normaloff>:/pic/cow32.png</iconset>
   </property>
   <property name="text">
    <string>Animal Action</string>
   </property>
   <property name="toolTip">
    <string>Animal Action</string>
   </property>
  </action>
  <action name="filterSaplAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res/res.qrc">
     <normaloff>:/pic/soiltesting32.png</normaloff>:/pic/soiltesting32.png</iconset>
   </property>
   <property name="text">
    <string>Soil Sample</string>
   </property>
   <property name="toolTip">
    <string>Soil Sample</string>
   </property>
  </action>
  <action name="filterMiscAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res/res.qrc">
     <normaloff>:/pic/comment.png</normaloff>:/pic/comment.png</iconset>
   </property>
   <property name="text">
    <string>Misc. Action</string>
   </property>
   <property name="toolTip">
    <string>Misc. Action</string>
   </property>
  </action>
  <action name="filterApplyAct">
   <property name="icon">
    <iconset resource="res/res.qrc">
     <normaloff>:/pic/reload.png</normaloff>:/pic/reload.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ActionsView</class>
   <extends>QTreeView</extends>
   <header>actionsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res/res.qrc"/>
 </resources>
 <connections/>
</ui>
